app/routes/task.py

```diff
@@ -0,0 +1,184 @@
+import datetime
+from flask import Blueprint
+from app.models.task import Task
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from dotenv import load_dotenv
+import requests
+import os
+
+
+bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response(
+            {"message": f"{cls.__name__} {model_id} invalid"}, 400))
+
+    model = cls.query.get(model_id)
+    if not model:
+        abort(make_response(
+            {"message": f"{cls.__name__} {model_id} not found"}, 404))
+
+    return model
+
+def validate_request(cls,data):
+    try:
+        new_cls = cls(title =data["title"],
+                description =data["description"])
+    except:
+        abort(make_response(
+            {"details": "Invalid data"}, 400))
+    return new_cls
+
+@bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    new_task = validate_request(Task,request_body)
+
+    db.session.add(new_task)
+    db.session.commit()
+
+    return make_response({
+            "task": {
+            "id":new_task.task_id,
+            "title": new_task.title,
+            "description": new_task.description,
+            "is_complete": bool(new_task.completed_at)
```

Comment:
Even though a new task is logically not going to be completed in the same request in which it's being created, I like how you're handling it here instead of hard-coding `is_complete = False`.